components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -8277,6 +8339,27 @@ components:
         advertiseOnly:
           type: boolean
           example: true
+    APIQuickInfo:
```

Comment:
Can't we reuse the existing `APIInfo` swagger model instead of introducing a new one. Will there any impact of having additional fields without values.

Reply:
Yeah it was suggested to use a new minimal DTO in the code review